pkg/apis/operator/v1alpha1/tektonconfig_validation.go

```diff
@@ -106,3 +136,39 @@ func isValueInArray(arr []string, key string) bool {
 	}
 	return false
 }
+
+func compareSCCAPriorityOverB(ctx context.Context, sccA, sccB string) (bool, error) {
```

Comment:
can this function be renamed  to some meaningful like compareMaxSCCPriorityOverDefaultSCC

Reply:
this takes 2 SCCs, SCCA and SCCB and it compares if SCCA has priority over SCCB - this is used for default SCC, maxAllowed SCC and for SCC requested in namespaces as well, which is why this is open ended for SCCA and SCCB instead of default and maxAllowed, does that make sense?